cilium/cmd/bpf_metrics_list.go

```diff
@@ -129,6 +165,44 @@ func listMetrics(bpfMetricsList map[string][]string) {
 	w.Flush()
 }
 
+func extractRow(key, value string) (*metricsRow, error) {
+	reasonCodeStr, directionCodeStr, ok := extractTwoValues(key)
+	if !ok {
+		return nil, fmt.Errorf("Cannot extract reason and traffic direction from map's key \"%s\"", key)
```

Comment:
Nit: error strings should start lower-cased

Source: https://github.com/golang/go/wiki/CodeReviewComments#error-strings